docs/test/bdd_instructions.md

```diff
@@ -1,4 +1,4 @@
-# Instruction to run bdd tests
+# Run bdd tests
```

Comment:
Why is this file still with the suffix _instructions? (I saw the others got renamed.)

Reply:
I missed the `test` folder. [Updated](https://github.com/hyperledger/aries-framework-go/compare/0f81218001bfede4a6d557d06ebae59d552475dd..9ce309499685bbe469cc977ba0746afd57600b88#diff-04c6e90faac2675aa89e2176d2eec7d8R48) the file names now.